recipe/meta.yaml

```diff
@@ -42,30 +41,30 @@ requirements:
     - qasync >=0.10.0
     - packaging
     - numpy
+    - importlib_metadata >=4.6  # [py<310]
+    - importlib_resources  # [py<39]
 
 test:
   imports:
-    - orangecanvas.main
-    - orangecanvas.application.canvasmain
+    - orangecanvas
   commands:
     - pip check
   requires:
     - pip
+    - typing-extensions
```

Comment:
I think this should be removed from here and added to the run section instead.

Reply:
got it! Thanks! I assumed it was a test dependency as it would cause tests to fail a bit earlier!